src/Integration.Vsix.UnitTests/SonarLintDaemon/SonarLintDaemonTests.cs

```diff
@@ -352,6 +354,73 @@ public void Dispose_WorkingDirectoryDeleted()
             testableDaemon.Dispose();
         }
 
+        [TestMethod]
+        public void DefaultVersionAndDownloadUrl()
```

Comment:
maybe mention in the name that the env var is not set (e.g. `DownloadUrlInEnvironmentVar_NotSet_UseDefault` )